packages/suite-desktop-core/e2e/tests/general/eap-modal.test.ts

```diff
@@ -11,13 +11,14 @@ import { test, expect } from '../../support/fixtures';
 // TODO: #15561 FIX settings cleanup:  eap setting is remembered even after cache cleanup at the beginning of the test. This shouldn't affect gha run but breaks the local one.
 test.skip(!process.env.GITHUB_ACTION, 'Test is working only in CI. Skipping local run.');
 
-test('Join early access button', async ({ dashboardPage, topBar, settingsPage }) => {
-    const buttonText = 'Leave';
+test.beforeAll(async ({ onboardingPage }) => {
+    await onboardingPage.completeOnboarding();
+});
```

Comment:
I tried to move this step to before in all tests but it didnt work in tests with emulators. So I will do the rest once I move emulator to fixtures